### DISCUSSION TITLE
Is there a postCSS plugin for the new @scope rule?

### DISCUSSION THREAD

**Discussion by SaroGFX** [csstools/postcss-plugins#1415] [Q&A] [answered]
I would imagine that the new [@scope](https://developer.mozilla.org/en-US/docs/Web/CSS/@scope#) rule can theoretically be polyfilled in the same way CSS modules is doing it. The only aspect is that it needs the correct reference from the source.

Does anything like this exist, or is it being worked on?

**Comment by romainmenke** [comment, marked as answer]
see : https://github.com/csstools/postcss-plugins/issues/1358

I am not personally working on it because of a lack of community funding for such an undertaking :)
The native scoping feature is very different from CSS modules in specific details and is much more complex to implement through static analysis. This is not an effort I want to do for free.

**Comment by SaroGFX** [comment]
Thanks for the reply @romainmenke ! Good to know :)

PS: I'll see if I can get work to show some [support](https://github.com/sponsors/csstools/).

**Reply by romainmenke** [reply]
Thank you @SaroGFX  🙇
